guides/v2.2/frontend-dev-guide/css-topics/css_debug.md

```diff
@@ -93,6 +93,20 @@ CSS source maps solve this issue. They help to find the `.less` file, where the
 
 ![node declaration autocomplete]({{ site.baseurl }}/common/images/fdg/with-map.png){:width="610px"}
 
+Magento 2 has a base set of Variables that define commonly used aspects of a theme. Colors, fonts, style of page titles, and so on are defined in these variables. 
```

Comment:
Please see my comments above.

Reply:
Hi @abrarpathan19 I don't see the edits reflected in v2.2. Everything else looks good.